.github/workflows/ci.yaml

```diff
@@ -25,8 +25,8 @@ jobs:
         bundler-cache: true
       timeout-minutes: 30
     - name: Set up Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v3
       with:
-        go-version: '1.17'
+        go-version-file: 'manageiq-operator/go.mod'
     - name: Run tests
       run: bundle exec rake
```

Comment:
Since this repo now has Go tests, should we run those as well?

Reply:
Done